### DISCUSSION TITLE
Documentation to prepare development environment

### DISCUSSION THREAD

**Discussion by ufoloko** [dnnsoftware/Dnn.Platform#5593] [General]
Hi Good day. Is there any documentation to prepare DNN Development environment. I really would like to write some code to collaborate with solution if possible.

I downloaded all git files to have a try, but honestly I cant found where to start.

**Comment by bdukes** [comment]
The main documentation is in https://github.com/dnnsoftware/Dnn.Platform/blob/develop/.github/BUILD.md. Hope it helps!

**Comment by bdukes** [comment]
There are also some videos at https://www.youtube.com/@stayathomednner6390/videos. They're a little out of date, but still quite informative.

**Comment by valadas** [comment]
Video documentation is great but does not age well, so when you have conflicting information our documentation on this repository (first link) prevails. I also do a weekly Dnn Open-Source co-coding session almost every Saturday on discord if you need some more hand-on help... https://discord.gg/48ta3hmt?event=1090353873635528764
